nrrd/tests/test_writing.py

```diff
@@ -177,17 +177,6 @@ def test_write_detached_raw_odd_extension(self):
         self.assertEqual(header['encoding'], 'raw')
         self.assertEqual('data file' in header, False)
 
-    def test_write_detached_raw_odd_extension(self):
-        output_data_filename = os.path.join(self.temp_write_dir, 'testfile_detached_raw.nrrd2')
```

Comment:
Was this test repeated? Sorry, couldn't understand the commit message.

Reply:
Yeah, exactly. I was confused at first too but I just copied/pasted the same test twice when writing and forgot to delete the duplicate.